integration_tests/test_gated.py

```diff
@@ -42,3 +42,9 @@ def test_gated_unknown_snap_failure(self):
         self.login()
         self.assertEqual(1, self.gated(
             'unknown', expected_error="Snap 'unknown' was not found"))
+
+    def test_gated_no_validations(self):
+        self.addCleanup(self.logout)
+        self.login()
+        self.assertEqual(0, self.gated(
+            'basic', expected_error="No validations for snap 'basic'"))
```

Comment:
Roberto,  "expected_error" sounds misleading. This text is not an error, right ?